src/components/RecipesList.js

```diff
@@ -33,7 +36,7 @@ class RecipesList extends React.Component {
         let recipes = data.map((recipe) => {
           return {id:recipe.id, title:recipe.title, image:recipe.image};
         });
-        this.setState({recipes: recipes})
+        this.setState({recipes: recipes, showDataLoading: false})
```

Comment:
Again could be a function call that switches true/ false based on what the current state

Reply:
Discussed above.